runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/io/DataflowDatastoreIOTest.java

```diff
@@ -1,66 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package org.apache.beam.runners.dataflow.io;
-
-import static org.apache.beam.sdk.transforms.display.DisplayDataMatchers.hasDisplayItem;
-
-import static org.hamcrest.Matchers.hasItem;
-import static org.junit.Assert.assertThat;
-
-import org.apache.beam.runners.dataflow.transforms.DataflowDisplayDataEvaluator;
-import org.apache.beam.sdk.io.datastore.DatastoreIO;
-import org.apache.beam.sdk.transforms.PTransform;
-import org.apache.beam.sdk.transforms.display.DisplayData;
-import org.apache.beam.sdk.transforms.display.DisplayDataEvaluator;
-import org.apache.beam.sdk.values.PBegin;
-import org.apache.beam.sdk.values.PCollection;
-import org.apache.beam.sdk.values.POutput;
-
-import com.google.datastore.v1beta3.Entity;
-import com.google.datastore.v1beta3.Query;
-
-import org.junit.Test;
-
-import java.util.Set;
-
-/**
- * Unit tests for Dataflow usage of {@link DatastoreIO} transforms.
- */
-public class DataflowDatastoreIOTest {
-  @Test
-  public void testSourcePrimitiveDisplayData() {
```

Comment:
Dataflow only changes pub sub. All the rest of these can be deleted safely.
It definitely does not change how Datastore is run and it never will.

Not sure why we think to need this test.
On Fri, Jul 8, 2016 at 06:10 lukecwik <notifications@github.com> wrote:

> In
> runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/io/DataflowDatastoreIOTest.java
> <https://github.com/apache/incubator-beam/pull/584#discussion_r70070789>:
>
> > -import org.apache.beam.sdk.values.PCollection;
> > -import org.apache.beam.sdk.values.POutput;
> > -
> > -import com.google.datastore.v1beta3.Entity;
> > -import com.google.datastore.v1beta3.Query;
> > -
> > -import org.junit.Test;
> > -
> > -import java.util.Set;
> > -
> > -/**
> > - * Unit tests for Dataflow usage of {@link DatastoreIO} transforms.
> > - */
> > -public class DataflowDatastoreIOTest {
> > -  @Test
> > -  public void testSourcePrimitiveDisplayData() {
>
> I think the consensus is that we leave the test as is and at some point in
> time in the future, display data will use @NeedsRunner annotation or when
> there is a runner independent graph construction then we will be able to
> migrate this to use it.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/apache/incubator-beam/pull/584/files/d9f5e3bdb641173fd776bde509d136f860f45616#r70070789>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/AAgIT9urnOERul11qwGHQR68BdZdBQn8ks5qTkxZgaJpZM4JDvGo>
> .
>